tracer/build/timeit/Samples.HttpMessageHandler/run.cmd

```diff
@@ -3,17 +3,22 @@ IF EXIST results_Samples.HttpMessageHandler.windows.net462.json DEL /F results_S
 IF EXIST results_Samples.HttpMessageHandler.windows.netcoreapp31.json DEL /F results_Samples.HttpMessageHandler.windows.netcoreapp31.json
 IF EXIST results_Samples.HttpMessageHandler.windows.net60.json DEL /F results_Samples.HttpMessageHandler.windows.net60.json
 
+echo *********************
+echo Installing timeitsharp
+echo *********************
+dotnet tool update -g timeitsharp
```

Comment:
As above
```suggestion
dotnet tool update -g timeitsharp -version 0.0.8
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3942/commits/17d8311bf60ddddcb92fd37238a55cb2630c940c